pom.xml

```diff
@@ -159,6 +185,11 @@
                     <target>1.8</target>
```

Comment:
c'est encore en java 8?

Reply:
Yes there isnt much incentive to bump the Java version here